### PR TITLE
ci: Test with latest reef devel image

### PR DESCRIPTION
**Pull request by travisn**
Testing with the latest Reef devel image to confirm if Rook CI will be working with v18.2.1

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by travisn** [comment]
Reopening for another test run

File: tests/manifests/test-cluster-on-pvc-encrypted.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ spec:
           requests:
             storage: 5Gi
   cephVersion:
-    image: quay.io/ceph/ceph:v17.2.6
+    image: quay.io/ceph/daemon-base:latest-reef-devel
```
Comment (rkachach) [inline tests/manifests/test-cluster-on-pvc-encrypted.yaml:17]: I think we should have some tag around this and some automated script to update the image on all the tests. WDYT?

Comment (travisn) [reply tests/manifests/test-cluster-on-pvc-encrypted.yaml:17]: Some canary tests run with v18, some v17.2.6, and the golang tests run with [these or other devel images](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_installer.go#L45-L52). The issue in this case is that we need the canary tests to run in our daily CI against devel images, and not just with the PRs against those released images.

**Comment by travisn** [comment]
Not needed anymore

**Pull request closed without merging**